torchtune/modules/low_precision/nf4_linear.py

```diff
@@ -18,27 +18,29 @@ class FrozenNF4Linear(nn.Linear):
     NF4Tensor as its weight. This class also freezes its ``weight`` parameter
     and is meant to be used as the base Linear layer for modeling
     use cases such as QLoRA where base model parameters are frozen.
-    NOTE: biases are currently not supported.
 
     Args:
         in_dim (int): input dimension
         out_dim (int): output dimension
         device (Optional[torch.device]): device to use for the underlying weight. If ``None``, uses the default
             device given by `torch.get_default_device()`.
+        bias (bool): whether to include bias in the original linear layer. Default: False
         **kwargs: any additional arguments to pass to the underlying Linear layer.
 
-    Raises:
-        RuntimeError: if ``bias`` is set to ``True``
     """
 
     def __init__(
-        self, in_dim: int, out_dim: int, device: Optional[torch.device] = None, **kwargs
+        self,
+        in_dim: int,
+        out_dim: int,
+        device: Optional[torch.device] = None,
+        bias: bool = False,
+        **kwargs,
     ):
-        if "bias" in kwargs and kwargs.pop("bias"):
-            raise RuntimeError("FrozenNF4Linear does not currently support biases!")
-
-        super().__init__(in_dim, out_dim, device=device, bias=False, **kwargs)
+        super().__init__(in_dim, out_dim, device=device, bias=bias, **kwargs)
```

Comment:
What is this super doing? Is it safe to assume it won't do anything with the bias we don't want?

Reply:
It's the same as before, just passing bias explicitly to nn.Linear init

Reply:
Why doesn't this create an extra parameter in the parent class? Does it create it and then we override those same variables in our init? I feel it would be safer to inherit from nn.Module

Reply:
Okay, nvm, I misunderstood what's going on in the init here.